src/extension/environment/manager.ts

```diff
@@ -23,7 +25,8 @@ export class EnvironmentManager {
     this.context.subscriptions.push(
       workspace.onDidChangeConfiguration(this.onDidChangeConfiguration.bind(this)),
     )
-    if (this.isEnabled()) {
+
+    if (!this.isEnabled()) {
```

Comment:
Oh, snap. I had another change that I decided to back out, but my CMD+Z seemed to have skipped the last one. My bad.

Reply:
The status bar item is permanent and now the primary way to select an environment which is why it's no longer disposed.

Reply:
Got it. If the user sets the config `runme.experiments.environments` to `false`, it will still appear until they reset vscode. So it's not "real time", but I wanted to check if that's the way we're handling other configurations of this kind!

Reply:
Yep, that's fine since it's an experiment. We'll likely revise this a few more times before considering a public release.